aggs-injectable_test.go

```diff
@@ -0,0 +1,156 @@
+package aggretastic
+
+import (
+	"errors"
+	"github.com/olivere/elastic"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"reflect"
+)
+
+func Call(function interface{}, params ...interface{}) (result []reflect.Value, err error) {
+	f := reflect.ValueOf(function)
+	if len(params) != f.Type().NumIn() {
+		err = errors.New("The number of params is not adapted.")
+		return
+	}
+	in := make([]reflect.Value, len(params))
+	for k, param := range params {
+		in[k] = reflect.ValueOf(param)
+	}
+	result = f.Call(in)
+	return
+}
+
+var _ = Describe("Injectable", func() {
+	ch1 := make(chan Aggregation, len(aggMap["Injectable"]))
+	ch2 := make(chan Aggregation, len(aggMap["Injectable"]))
+	ch3 := make(chan Aggregation, len(aggMap["Injectable"]))
+	ch4 := make(chan Aggregation, len(aggMap["Injectable"]))
+	ch5 := make(chan Aggregation, len(aggMap["Injectable"]))
+	ch6 := make(chan Aggregation, len(aggMap["Injectable"]))
+	ch7 := make(chan Aggregation, len(aggMap["Injectable"]))
```

Comment:
Tests should be simple and easy-to-follow thing. Tests should be declarative.

Not very intuitive - what these 7 channels are for. So, every `It` will require creation of a new channel?

Reply:
My fault - this was temporary code, forgot to remove it